src/writer/blocks/ifelse.py

```
@@ -0,0 +1,87 @@
+                    lambda entry: self.runner.session.session_state.add_log_entry(
+                        "info", "Captured stdout", entry
+                    )
+                ),
+                use_logging_redirect(
+                    lambda entry: self.runner.session.session_state.add_log_entry(
+                        "info", "Captured logs", entry
+                    )
+                ),
+            ):
+                # Evaluate the expression and set the result
+                self.result = eval(expression, block_globals)
```

Ruff (0.14.5)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)